examples/00-systemcoupling/oscillating_plate.py

```diff
@@ -196,37 +191,74 @@
 force_transfer.option = "UsingExpression"
 force_transfer.value = "vector(5.0 [N], 0.0 [N], 0.0 [N]) if Time < 0.5 [s] else force"
 
-syc.setup.solution_control.time_step_size = 0.1
-syc.setup.solution_control.end_time = (
-    3.0  # shorten the run a bit, full run is 10 seconds
-)
+# %%
+# Time step size, end time, output controls
+syc.setup.solution_control.time_step_size = "0.1 [s]"  # time step is 0.1 [s]
+syc.setup.solution_control.end_time = 10  # end time is 10.0 [s]
 
 syc.setup.output_control.option = "EveryStep"
+syc.setup.output_control.generate_csv_chart_output = True
 
-print(f"SyC server info: {syc._native_api.GetServerInfo()}")
-
-# solve the coupled analysis
+# %%
+# Solve the coupled analysis
+# -------------------------------------
 syc.solution.solve()
 
-mapdl.finish()
 
-# post-process structural results
-mapdl.post1()
+# %%
+# Post-process the coupled analysis
+# -------------------------------------
 
-"""
-mapdl.result.animate_nodal_displacement(
-    rnum = 0,
-    loop=True,
-    add_text=False,
-    displacement_factor=1.0,
+# %%
+# Post-process the structural results
+mapdl.finish()
+mapdl.post1()
+mapdl.result.plot_nodal_displacement(
```

Comment:
I'm guessing that this is what is making the build fail currently.

It looks like pymapdl uses `xvfb-run` in its doc build (see `"Build documentation"` step in their `.github/workflows/.ci.yml`). It might need to be included as a dependency in `pyproject.toml` or something.